FrameworkSystem/private/standardLogging/gLogging.py

```diff
@@ -0,0 +1,335 @@
+"""
+gLogging
+"""
+
+__RCSID__ = "$Id$"
+
+import logging
+
+from DIRAC.FrameworkSystem.private.standardLogging.LogLevels import LogLevels
+
+from DIRAC.FrameworkSystem.private.standardLogging.Backend.StdoutBackend import StdoutBackend
+from DIRAC.FrameworkSystem.private.standardLogging.Backend.StderrBackend import StderrBackend
+from DIRAC.FrameworkSystem.private.standardLogging.Backend.FileBackend import FileBackend
+from DIRAC.FrameworkSystem.private.standardLogging.Backend.RemoteBackend import RemoteBackend
+
+
+class gLogging(object):
+  """
+  gLogging is a wrapper of the logger object from the standard logging library which integrate
+  some DIRAC concepts. It is the equivalent to the old gLogger object.
+
+  It is used like an interface to use the logger object of the logging library. 
+  Its purpose is to replace transparently the old gLogger object in the existing code in order to 
+  minimize the changes. 
+
+  In this way, each gLogging embed a logger of logging. It is possible to create sublogger,
+  set and get the level of the embedded logger and create log messages with it.
+
+  gLogging could delegate the initialization and the configuration to a factory of the root logger be it can not
+  because it has to wrap the old gLogger.  
+  """
+
+  _componentName = "Framework"
+
+  def __init__(self, father=None, fathername='', name=''):
+    """
+    Initialization of the gLogging object.
+    :params fathername: string representing the name of the father logger in the chain.
+    :params name: string representing the name of the logger in the chain. 
+    :params father: gLogging father of this new gLogging.
+    By default, 'fathername' and 'name' are empty, because getChild accepts only string and the first empty
+    string corresponds to the root logger. 
+
+    Example: 
+    logging.getLogger('') == logging.getLogger('root') root logger
+    logging.getLogger('root').getChild('log') == root.log == log child of root
+    """
+
+    # gLogging chain
+    self._childrens = []
+    self._parent = father
+
+    # all the different backends
+    self._backendsDict = {'stdout': StdoutBackend(),
+                          'stderr': StderrBackend(),
+                          'file': FileBackend(),
+                          'server': RemoteBackend()}
+
+    # initialize display options with the ones of the gLogging parent
+    if self._parent is not None:
+      self._options = self._parent.getDisplayOptions()
+    else:
+      self._options = {'showHeaders': True, 'showThreads': False, 'Color': False, 'Path': False}
+
+    # dictionary of the option state, modified by the user or not
+    # this is to give to the options the same behaviour that the logging setLevel()
+    self._optionsmodified = {'showHeaders': False, 'showThreads': False}
```

Comment:
I changed to camel style and I added a comment to explain the idea. Do you understand now ?